src/service/conversation-manager-service.ts

```diff
@@ -340,7 +341,7 @@ export class ConversationManagerService {
     contact: Contact;
   }[] {
     return Array.from(this.conversationWithContactByConversationId.values())
-      .filter(({ conversation }) => conversation.status === "pending")
+      .filter(({ conversation }) => conversation.status === "receive-pending")
       .map(({ conversation, contact }) => ({
         conversation: conversation as PendingConversation,
         contact,
```

Comment:
should we rename `getReceivedPendingConversationWithContact` instead to better reflect the applied filter?